bundles/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/SenecHomeHandlerFactory.java

```diff
@@ -37,15 +38,17 @@
 @NonNullByDefault
 @Component(configurationPid = "binding.senechome", service = ThingHandlerFactory.class)
 public class SenecHomeHandlerFactory extends BaseThingHandlerFactory {
+    public static final String BINDING_ID = "senechome";
```

Comment:
This constant is already defined in SenecHomeBindingConstants.

Reply:
removed this additional constant